datasets/norne/README.md

```diff
@@ -63,7 +63,80 @@ NorNE ads named entity annotations on top of the Norwegian Dependency Treebank.
 
 ### Languages
 
-Both Norwegian Bokmål (`bookmal`) and Nynorsk (`nynorsk`) are supported as different configs in this dataset.
+Both Norwegian Bokmål (`bookmal`) and Nynorsk (`nynorsk`) are supported as different configs in this dataset. An extra config for the combined languages is also included (`combined`).
+
+## Dataset Structure
+
+Each entry contains text sentences, their language, identifiers, tokens, lemmas, and corresponding NER and POS tag lists.
+
+### Data Instances
+
+An example of the `train` split of the `bokmaal` config.
+
+<table class="dataframe" border="1">
```

Comment:
Here we expect an example from the actual dataset object in python.

You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]` for eample

Reply:
Done.